src/Package/Impl/ProjectSystem/RProjectLoadHooks.cs

```diff
@@ -72,7 +72,8 @@ internal sealed class RProjectLoadHooks {
             , IRToolsSettings toolsSettings
             , IThreadHandling threadHandling
             , ISurveyNewsService surveyNews
-            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider) {
+            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider
+            , ICoreShell coreShell) {
```

Comment:
Yes, except one `GetGlobalService`. I can either a) import AppShell instead or b) push `GetGlobalService` down to core.